lib/base_service.ts

```diff
@@ -220,12 +294,14 @@ export class BaseService {
     const _username: string = process.env[`${_name}_USERNAME`] || process.env[`${_nameWithUnderscore}_USERNAME`];
     const _password: string = process.env[`${_name}_PASSWORD`] || process.env[`${_nameWithUnderscore}_PASSWORD`];
     const _apiKey: string = process.env[`${_name}_API_KEY`] || process.env[`${_nameWithUnderscore}_API_KEY`];
+    const _platformApiKey: string = process.env[`${_name}_PLATFORM_API_KEY`] || process.env[`${_nameWithUnderscore}_PLATFORM_API_KEY`];
```

Comment:
Just a note that this environment variable should match what's being provided in `VCAP_SERVICES` (see German's pull request -- I think it may be just `apikey` right now).

Reply:
Someone correct me if I am wrong but I believe these are for pulling in from standard environment variables, while there is another function (`getCredentialsFromBluemix`) for pulling in the `VCAP_SERVICES` credentials.